blacktie/jatmibroker-xatmi/src/main/java/org/jboss/narayana/blacktie/jatmibroker/core/transport/OrbManagement.java

```diff
@@ -26,9 +26,9 @@
 public class OrbManagement implements Runnable {
     private static final Logger log = LogManager.getLogger(OrbManagement.class);
     private static final String CorbaOrbClassProp = "org.omg.CORBA.ORBClass";
-    private static final String CorbaOrbClassValue = "org.jacorb.orb.ORB";
+    private static final String CorbaOrbClassValue = "com.sun.corba.se.impl.orb.ORBImpl";
```

Comment:
I agree that we no longer need OrbManagement. The test I referred to in yesterdays meeting uses JTS to propagate a transaction initiated by BlackTie between two servers but that is different from requiring BlackTie to support OTS - so my apologies for introducing the confusion.

So, although we initialize the OrbManagement function, we never actually use CORBA anywhere in the BlackTie code base. So it is up to you whether you want to make the change here or simply create a new JIRA for removing CORBA and fix it another time.